### PR TITLE
V1 to v2 doc

### PR DESCRIPTION
**Pull request by navalev**

### CODE REVIEW

File: docs/presidio_V2.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Presidio Revamp (aka V2)
+
+As of March 2021, Presidio had undergo a revamp to a new version refereed to as **V2**.
+
+The main changes introduced in **V2** are:
+
+1. gRPC replaced with HTTP to allow more customizable APIs and easier debugging
+2. Presidio Anonymizer is Python based and pip installable.
+3. Focus on the Analyzer and Anonymizer services. Other services are deprecated and potentially be migrated over time to **V2** with the help of the community.
```
Comment (shiranr) [inline docs/presidio_V2.md:9]:
```suggestion
3. Focus on the Analyzer and Anonymizer services. 

Other services are deprecated and potentially be migrated over time to **V2** with the help of the community.

```

File: docs/presidio_V2.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+The main changes introduced in **V2** are:
+
+1. gRPC replaced with HTTP to allow more customizable APIs and easier debugging
+2. Presidio Anonymizer is Python based and pip installable.
+3. Focus on the Analyzer and Anonymizer services. Other services are deprecated and potentially be migrated over time to **V2** with the help of the community.
+4. Improved documentation, samples and build flows.
+5. Presidio Analyzer - removed use of templates and external recognizer store
+
+Notes:
+The legacy V1 code base will continue to be available under branch [V1](https://github.com/microsoft/presidio/tree/V1) but will no longer be officially supported.
```
Comment (shiranr) [inline docs/presidio_V2.md:14]: we need to add a todo to verify this link works in the future.

File: docs/presidio_V2.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+## API Changes
+
+The move from gRPC to HTTP based APIs included changes to the API requests.
+
+1. Moving from GRPC to HTTP - moving from structures to jsons.
```
Comment (shiranr) [inline docs/presidio_V2.md:20]:
```suggestion
1. Change in payload - moving from structures to jsons.

```

File: docs/presidio_V2.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+      },
+      "location": {
+        "start": 24,
+        "end": 32,
+        "length": 6
+      },
+      "score": 0.8
+    }
+  ]
+}
```
Comment (shiranr) [inline docs/presidio_V2.md:111]:
```suggestion
{
  "text": "hello world, my name is Jane Doe. My number is: 034453334",
  "template": {
    "description": "DEPRECATED",
    "create_time": "DEPRECATED",
    "modified_time": "DEPRECATED",
    "default_transformation": {
      "replace_value": {
        ...
      },
      "redact_value": {
        ...
      },
      "hash_value": {
        ...
      },
      "mask_value": {
        ...
      },
      "fpe_value": {
        ...
      }
    },
    "field_type_transformations": [
      {
        "fields": [
          {
            "name": "FIRST_NAME",
            "min_score": "0.2"
          }
        ],
        "transfomarion": {
          "replace_value": {...},
          "redact_value": {...},
          "hash_value": {...},
          "mask_value": {...},
          "fpe_value": {...}
        }
      }
    ],
    "analyze_results": [
      {
        "text": "Jane",
        "field": {
          "name": "FIRST_NAME",
          "min_score": "0.5"
        },
        "location": {
          "start": 24,
          "end": 32,
          "length": 6
        },
        "score": 0.8
      }
    ]
  }

```

Comment (shiranr) [reply docs/presidio_V2.md:111]: The json was invalid, I changed the commas and brackets to fix it.

**Review by shiranr — COMMENTED** [review]
Looks good some small suggestions.

File: docs/presidio_V2.md (hunk 1 above)
Comment (navalev) [reply docs/presidio_V2.md:9]: not sure this should be a seperate comment

Comment (shiranr) [reply docs/presidio_V2.md:9]: it's not separate comment, just a new line :)

Comment (shiranr) [reply docs/presidio_V2.md:9]: but it's a nit

**Review by shiranr — APPROVED** [review]

File: docs/presidio_V2.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
+# Presidio Revamp (aka V2)
+
+As of March 2021, Presidio had undergo a revamp to a new version refereed to as **V2**.
+
+The main changes introduced in **V2** are:
+
+1. gRPC replaced with HTTP to allow more customizable APIs and easier debugging
+2. Presidio Anonymizer is Python based and pip installable.
+3. Focus on the Analyzer and Anonymizer services. Other services are deprecated and potentially be migrated over time to **V2** with the help of the community.
```
Comment (omri374) [inline docs/presidio_V2.md:9]: Consider mentioning the image redactor

File: docs/presidio_V2.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+The legacy V1 code base will continue to be available under branch [V1](https://github.com/microsoft/presidio/tree/V1) but will no longer be officially supported.
+
+## API Changes
+
+The move from gRPC to HTTP based APIs included changes to the API requests.
+
+1. Change in payload - moving from structures to jsons.
+
+2. Removing templates from the API - includes flattening the json.
+3. Using snake_case instead of camel_case.
```
Comment (omri374) [inline docs/presidio_V2.md:24]: nit: camelCase 😊

File: docs/presidio_V2.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+
+Bellow is a detailed outline of all the changes done to the Analyzer and Anonymizer.
```
Comment (omri374) [inline docs/presidio_V2.md:26]: Typo: Below

File: docs/presidio_V2.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+
+```json
+{
+    "text": "My phone number is 212-555-5555",
+    "entities": ["PHONE_NUMBER"],
+    "language": "en",
+    "correlation_id": "213",
+    "score_threshold": 0.5,
+    "trace": true,
+    "remove_interpretability_response": true
```
Comment (omri374) [inline docs/presidio_V2.md:63]: Whoever merges first should update this to return_decision_process 😊

Comment (navalev) [reply docs/presidio_V2.md]: done

File: docs/presidio_V2.md (hunk 5 above)
Comment (navalev) [reply docs/presidio_V2.md]: added a note on the image-redactor

**Review by omri374 — APPROVED** [review]

**Pull request merged**